### PR TITLE
feat: some cleanup for making a3 easier

### PR DESCRIPTION
**Pull request by reginawang99**
- more descriptive reviews
- bring some animation text up
- fix final number stars
- removed gaussian distribution
- made game easier

next: 
- more descriptive reviews (whether to increase or decrease a value)
- \# reviews corresponding to \# days ab testing

### CODE REVIEW

File: src/components/Activities/Activity3/Game/DebuggingResults.tsx [hunk 1]
```diff
@@ -23,27 +23,25 @@ function DebuggingResults(): JSX.Element {
...
+      buttonText: 'Change Priorities',
       onClick: improveRecs,
-      dayCost: 3,
+      daysMin: 3,
     },
     'No change': {
       buttonText: 'Continue to A/B Testing',
       onClick: goNextState,
-      dayCost: 0,
+      daysMin: -10,
```
Comment (tr89on) [inline src/components/Activities/Activity3/Game/DebuggingResults.tsx:44]: Why is daysMin negative? Does daysLeft ever end up < 0?

Comment (reginawang99) [reply src/components/Activities/Activity3/Game/DebuggingResults.tsx:44]: it just in casse there are any bugs!

Comment (tr89on) [reply src/components/Activities/Activity3/Game/DebuggingResults.tsx:44]: cool cool

**Review by tr89on — APPROVED** [review]
lgtm

**Pull request merged**